YesterdayApi/Core/Identity/Validators/UserIdentityRequestValidator.cs

```diff
@@ -0,0 +1,15 @@
+using FluentValidation;
+using YesterdayApi.Core.Identity.Web;
+using YesterdayApi.Utilities.AutomaticDI;
+
+namespace YesterdayApi.Core.Identity.Validators
+{
+    public class UserIdentityRequestValidator : AbstractValidator<UserIdentityRequest>, ITransient
+    {
+        public UserIdentityRequestValidator()
+        {
+            RuleFor(ur => ur.Password).NotNull();
+            RuleFor(ur => ur.Password).Length(5, 50);
```

Comment:
Correct me if I'm wrong but I think you can put two requirements in one line like:

```c#
RuleFor(ur => ur.Password).NotNull().Length(5, 50);
```

Reply:
Idk i think it's better to separate separate rules in different lines so they don't get mixed up.